system/jlib/jstats.cpp

```diff
@@ -1994,6 +2015,15 @@ class CRootStatisticCollection : public CStatisticCollection
         out.append(creator);
         out.append(whenCreated);
     }
+    virtual void mergeInto(IStatisticGatherer & target) const override
+    {
+//        StatsOptScope block(target, id);
```

Comment:
Should change to a comment
// Similar to CStatisticCollection::mergeInfo but do not add the root scope.
the rest of the function could be a shared helper function in the base, but happy for it to remain a clone.